test/docs/jdk/javadoc/doccheck/doccheckutils/checkers/LinkChecker.java

```diff
@@ -223,13 +223,14 @@ public boolean isOK() {
         return duplicateIds == 0
                 && missingIds == 0
                 && missingFiles == 0
-                && badSchemes == 0;
+                && badSchemes == 0
+                && log.noErrors();
```

Comment:
So we have both these counters and the `log` instance to track errors. Shouldn't the `missingIds` counter have bee increased when the error was logged?

Reply:
Indeed, would be much better if we can just check `log.noErrors()`.